main_test.go

```diff
@@ -93,134 +87,117 @@ func failMainTest(err string) {
 	fmt.Println(err) //nolint:forbidigo
 }
 
-func startTestServer() (func(), bool) {
-	dir, err := os.MkdirTemp("", "ibackup-test")
-	if err != nil {
-		failMainTest(err.Error())
-
-		return nil, false
-	}
-
-	os.Setenv("XDG_STATE_HOME", dir)
+var servers []*TestServer //nolint:gochecknoglobals
+
+type TestServer struct {
```

Comment:
Likewise, the type and associated funcs declared before the variable of the type.